### PR TITLE
Fix: Display Weather Information based on User location

### PR DESCRIPTION
**Pull request by AkhileshManda**
After addition of some features the app only took user location as input but did not show the related weather information, it just kept saying loading.

The PR fixes the following bug.

image

### CODE REVIEW

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/screen/home.js [hunk 1]
```diff
@@ -64,9 +64,20 @@ function App() {
...
+
+        setIsLoaded(true);
+        console.log("name",result.name);
+        setCity(`${result.name}, ${result.sys.country}`);
+        setResults(results);
+      })
+      .catch((err)=>{
+        setIsLoaded(false);
+        setError(err);
+      });;
```
[the remark this answers is not in the document]
Comment (AkhileshManda) [reply src/screen/home.js]: Done

**Comment by arpitBhalla** [comment]
> After addition of some features the app only took user location as input but did not show the related weather information, it just kept saying loading.

Might be some issue on your end or maybe API key something? [The preview](https://prep-22-jul-prep-2-project.netlify.app/) works fine.

**Comment by AkhileshManda** [comment]
image
this is what I am getting when I run the deploy link

File: src/Components/Attractions.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const Attractions = (results) => {
+	// console.log(results.results.city.name);
+	const locationData = results.results;
+	const city = locationData.city.name;
+	const lng = locationData.city.coord.lon;
+	const lat = locationData.city.coord.lat;
+
+	useEffect(() => {
+		fetch(`https://api.opentripmap.com/0.1/en/places/radius?radius=1000&lon=${lng}&lat=${lat}9&rate=2&format=json&limit=15&apikey=5ae2e3f221c38a28845f05b6c06a2c73eb6f2499d3b1881719d6dd39`)
```
Comment (arpitBhalla) [inline src/Components/Attractions.js:11]: Maybe use this instead `{process.env.REACT_APP_APIKEY}`

File: src/Components/Attractions.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+			.then(res => res.json())
+			.then(data => {
+				console.log(data);
+			})
```
Comment (arpitBhalla) [inline src/Components/Attractions.js:15]: Are we using this API's response?

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/Components/Attractions.js (hunk 3 above)
Comment (AkhileshManda) [reply src/Components/Attractions.js:15]: Yes, this is for our next assigned issue showing attractions....I pushed the changes to the main branch by mistake.

Comment (arpitBhalla) [reply src/Components/Attractions.js:15]: Cool, revert these changes

Comment (AkhileshManda) [reply src/Components/Attractions.js]: Done!

**Pull request merged**